examples/typescript/cdk8s-plus-elasticsearch-query/main.test.ts

```diff
@@ -0,0 +1,224 @@
+import { MyChart } from "./main";
+import { Testing } from "cdk8s";
+
+describe("Placeholder", () => {
+  test("Empty", () => {
+    const app = Testing.app();
+    const chart = new MyChart(app, "test-chart");
+    const results = Testing.synth(chart);
+    expect(results).toMatchInlineSnapshot(`
```

Comment:
Prefer external snapshots for big things like that

Reply:
👍